mongo_connector/oplog_manager.py

```diff
@@ -74,11 +75,14 @@ def __init__(self, primary_client, doc_managers,
         # The set of namespaces to process from the mongo cluster.
         self.namespace_set = kwargs.get('ns_set', [])
 
+        # The set of namespaces not to process from the mongo cluster.
+        self.ex_namespace_set = kwargs.get('ex_ns_set', [])
+
         # The set of gridfs namespaces to process from the mongo cluster
         self.gridfs_set = kwargs.get('gridfs_set', [])
 
-        # The dict of source namespaces to destination namespaces
-        self.dest_mapping = kwargs.get('dest_mapping', {})
+        # The structure handling dest_mappings dynamically
+        self.dest_mapping = dest_mapping_stru
```

Comment:
Sorry, I dont get your point. The old dest_mapping will never be used and I passed the new DestMapping object to the OplogThread init function. I used the original variable name self.dest_mapping just because I can avoid to change all the occurrences of self.dest_mapping.get() function with new names. If you think this makes people confused, then I change all the self.dest_mapping to self.dest_mapping_stru, but do you think we need to keep the original self.dest_mapping even it is not used anymore?